backbone/src/main/java/org/researchstack/backbone/model/survey/SurveyItem.java

```diff
@@ -54,4 +51,30 @@ public int compare(SurveyItemType lhs, SurveyItemType rhs) {
             return 0;
         }
     }
+
+    @Override
+    public int hashCode() {
+        if (identifier == null) {
```

Comment:
Those are all good thoughts.  

I also think that "identifier" was meant to the on unique item, and I believe that is enforced on the bridge study manager.

After reflecting more, I believe the best approach here would be to keep the current implementation, and then address adding more fields when we implement the load survey task.  Because, once that is implemented, we should have a better feel for the usefulness of loading SurveyItems, saving them in the DB, and who knows, maybe we don't end up even saving the SurveyItems but saving the produced Task/Step relationships instead.  Does that sound like a good approach?  If so, I can add some TODO: comments here and maybe you can add some in the Bridge API to address this.